ninja-core/src/site/markdown/documentation/basic_concepts/controllers.md

```diff
@@ -125,7 +125,10 @@ If your custom object has a date/timestamp field, first make sure that you use t
 <a href="http://docs.oracle.com/javase/7/docs/api/java/util/Date.html">java.util.Date</a> 
 type and when you pass some date to the controller pass in the 
 <a href="http://joda-time.sourceforge.net/api-release/org/joda/time/format/ISODateTimeFormat.html#localDateOptionalTimeParser%28%29">format</a> 
-accepted by Joda Time library.
+accepted by Joda Time library. Or you can register a <code>ninja.params.ParamParser</code> with your own date format parsing in your <code>conf.Module</code>:
+<pre class="prettyprint">
+ParamParsers.registerParamParser(Date.class, MyDateParamParser.class);
```

Comment:
Not sure. I think it's quite the same discussion as the NinjaProperties given to a module as constructor arg... If it is not static, you'll not be able too to inject it in the conf.Module. So where will users do the early job of registering parsers? 

Not that, with the way I did the PR, at least all custom parsers will now be Guice-aware.